test/file.js

```diff
@@ -118,7 +115,7 @@ describe('/file', () => {
               if (err) {
                 throw err;
               }
-              assert.ok(res.body.url.indexOf('http://localhost/file/') !== -1);
+              assert.ok(res.body.url.indexOf('https://localhost/file/') !== -1);
```

Comment:
Not a blocker, but this url isn't set in the test file anywhere, so is testing *current* configuration, which makes it liable to be fragile if config changes in future.

I'd probably want to explicitly set the config value in the test to make it robust.